superset-frontend/src/SqlLab/types.ts

```diff
@@ -82,3 +97,31 @@ export interface QueryEditor {
     errors: SupersetError[];
   };
 }
+
+// state refers to superset-frontend/src/components/MessageToasts/getToastsFromPyFlashMessages.js and superset-frontend/src/components/MessageToasts/actions.ts
```

Comment:
I think the code explains well enough without needing this comment, could you remove this?

Reply:
removed. Thank you